### PR TITLE
Add 7.10 branches to conf.yaml

### PR DESCRIPTION
**Pull request by lcawl**
This PR adds new 7.10 branches to the appropriate books.

NOTE: These build checks will fail until all the branches exist.

### CODE REVIEW

File: conf.yaml [hunk 1]
```diff
@@ -1768,8 +1768,8 @@ contents:
           - title:      Ingest Management Guide
             prefix:     en/ingest-management
             current:    *stackcurrent
-            branches:   [ master, 7.x, 7.9, 7.8 ]
-            live:       [ master, 7.x, 7.9, 7.8 ]
+            branches:   [ master, 7.x, 7.10, 7.9, 7.8 ]
+            live:       [ master, 7.x, 7.10, 7.9, 7.8 ]
```
Comment (gtback) [inline conf.yaml:1772]: Should we take 7.8 out of this list? Or can we change it to `*stacklive`?

**Review by gtback — COMMENTED** [review]
I also noticed we didn't add 7.10 to the Enterprise Search or some of the APM books. Was that intentional? Did you just add 7.10 whenever there was a 7.x?

File: conf.yaml (hunk 1 above)
Comment (lcawl) [reply conf.yaml:1772]: I think this is a question for @dedemorton.  Can happen separately from this PR (before or after) as far as I'm concerned.

**Comment by lcawl** [comment]
> Did you just add 7.10 whenever there was a 7.x?

Yup, if there are other exceptional cases that don't have a 7.x, we can follow up after this PR if necessary. If they don't have a 7.x there must not be links to those books from other books in the library.

File: conf.yaml (hunk 1 above)
Comment (dedemorton) [reply conf.yaml:1772]: @lcawl I think I did that before because I didn't have all the branches in the list. You should be able to use *stacklive now.

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs

**Comment by lcawl** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by jmikell821 — APPROVED** [review]

**Pull request merged**